Detectors/PHOS/simulation/src/Detector.cxx

```diff
@@ -795,120 +775,20 @@ void Detector::ConstructEMCGeometry()
 
   // Put created EMC geometry into PHOS volume
 
-  z = geom->GetCPVBoxSize(1) / 2.;
-  // normal PHOS module
-  fMC->Gspos("PEMC", 1, "PHOS", 0., 0., z, 0, "ONLY");
-  if (mCreateCPV) // Module with CPV
-    fMC->Gspos("PEMC", 1, "PHOC", 0., 0., z, 0, "ONLY");
+  z = geom->GetDistATBtoModule()-geom->GetPHOSATBParams()[3]+geom->GetEMCParams()[3];
+  
+  //PHOS AirTightBox
+  fMC->Gsvolu("PATB", "TRD1", getMediumID(ID_AIR), geom->GetPHOSATBParams(), 4);
   if (mCreateHalfMod) // half of PHOS module
-    fMC->Gspos("PEMH", 1, "PHOH", 0., 0., z, 0, "ONLY");
-}
-//-----------------------------------------
-void Detector::ConstructCPVGeometry()
-{
-  // Create the PHOS-CPV geometry for GEANT
-  // Author: Yuri Kharlov 11 September 2000
-  // Adopted for O2 project 2017
-
-  phos::GeometryParams* geom = phos::GeometryParams::GetInstance();
-
-  Float_t par[3] = { 0 }, x = 0., y = 0., z = 0.;
-
-  // The box containing all CPV for one PHOS module filled with air
-  par[0] = geom->GetCPVBoxSize(0) / 2.0;
-  par[1] = geom->GetCPVBoxSize(1) / 2.0;
-  par[2] = geom->GetCPVBoxSize(2) / 2.0;
-  fMC->Gsvolu("PCPV", "BOX ", getMediumID(ID_AIR), par, 3);
-
-  const Float_t* emcParams = geom->GetEMCParams();
-  z = -emcParams[3];
-  Int_t rotm;
-  Matrix(rotm, 90., 0., 0., 0., 90., 90.);
-
-  fMC->Gspos("PCPV", 1, "PHOC", 0.0, 0.0, z, rotm, "ONLY");
-
-  // Gassiplex board
-
-  par[0] = geom->GetGassiplexChipSize(0) / 2.;
-  par[1] = geom->GetGassiplexChipSize(1) / 2.;
-  par[2] = geom->GetGassiplexChipSize(2) / 2.;
-  fMC->Gsvolu("PCPC", "BOX ", getMediumID(ID_TEXTOLIT), par, 3);
-
-  // Cu+Ni foil covers Gassiplex board
-
-  par[1] = geom->GetCPVCuNiFoilThickness() / 2;
-  fMC->Gsvolu("PCPD", "BOX ", getMediumID(ID_CUPPER), par, 3);
-  y = -(geom->GetGassiplexChipSize(1) / 2 - par[1]);
-  fMC->Gspos("PCPD", 1, "PCPC", 0, y, 0, 0, "ONLY");
-
-  // Position of the chip inside CPV
+    fMC->Gsvolu("PATH", "TRD1", getMediumID(ID_AIR), geom->GetPHOSATBParams(), 4);
```

Comment:
Formatting conventions: braces must be used even if body of control construct consists of only one statement